code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -367,3 +371,21 @@
 	backpack_contents = list(/obj/item/storage/box/syndie=1,\
 		/obj/item/tank/jetpack/oxygen/harness=1,\
 		/obj/item/gun/ballistic/automatic/pistol=1)
+
+/datum/game_mode/nuclear/generate_credit_text()
+	var/list/round_credits = list()
+	var/len_before_addition
+
+	if((world.time-SSticker.round_start_time) < (FLUKEOPS_TIME_DELAY)) // If the nukies died super early, they're basically a massive disappointment
```

Comment:
Just because the round ended within 20 minutes, doesn't mean the nukeops lost. Would be great if they only became flukeops if they did lose. With that said, I'm not sure if the 20 minute mark is a good way to mark the round as flukeops at all. Maybe all non-war nukies that lose should be fluke ops?

Reply:
Way over-engineering the problem here.